### PR TITLE
Publish soljson and alpine static build on github release page.

### PR DESCRIPTION
**Pull request by chriseth**
This still needs to be tested using an actual release.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -176,7 +176,12 @@ install:
     - test $SOLC_INSTALL_DEPS_TRAVIS != On || (scripts/install_deps.sh)
     - test "$TRAVIS_OS_NAME" != "linux" || (scripts/install_cmake.sh)
     - echo -n "$TRAVIS_COMMIT" > commit_hash.txt
-    - test $SOLC_DOCKER != On || (docker build -t ethereum/solc:build -f scripts/Dockerfile .)
+    - test $SOLC_DOCKER != On || (
+        docker build -t ethereum/solc:build -f scripts/Dockerfile .
+        tmp_container=$(docker create brainbot-com/solidity-static sh)
```
Comment (konradkonrad) [inline .travis.yml]: this should probably read:
```
tmp_container=$(docker create ethereum/solc:build sh)
```

File: .travis.yml [hunk 2]
```diff
@@ -176,7 +176,12 @@ install:
...
+        mkdir -p upload
+        docker cp ${tmp_container}:/usr/local/bin/solc upload/
```
Comment (konradkonrad) [inline .travis.yml]: make this
```
docker cp ${tmp_container}:/usr/bin/solc upload/
```
-- your binary ends up in `/usr/bin`:
```
 docker run --rm -it ethereum/solc:stable sh -c 'which solc'
/usr/bin/solc
```

**Review by konradkonrad — CHANGES_REQUESTED** [review]
thanks for tackling this! there are two minor issues that I could spot right away.

File: .travis.yml (hunk 1 above)
Comment (axic) [reply .travis.yml]: Shouldn't this part be made into a script instead?

**Comment by chriseth** [comment]
Tested this and it is working: https://github.com/chriseth/solidity/releases/tag/v0.0.3 (look at the nice small solc executable :-) )

**Comment by chriseth** [comment]
@axic can this be merged?

**Comment by axic** [comment]
Is there a way to name it a bit more expressively? It won't be mistaken for a Windows binary because lacking `.exe`, but it could be mistaken for an macOS binary.

Perhaps `solc-static-linux`?

File: .travis.yml [hunk 3]
```diff
@@ -223,11 +223,8 @@ deploy:
...
       file_glob: true
-      file:
-          - $TRAVIS_BUILD_DIR/solidity*.zip
-          - $TRAVIS_BUILD_DIR/solidity*tar.gz
+      file: $TRAVIS_BUILD_DIR/upload/*
       skip_cleanup: true
       on:
           all_branches: true
           tags: true
-          condition: $SOLC_RELEASE == On
```
Comment (axic) [inline .travis.yml:233]: Is this intentional?

**Comment by axic** [comment]
Otherwise looks good!

File: .travis.yml (hunk 3 above)
Comment (chriseth) [reply .travis.yml:233]: Yes, it will just attach everything that is in the `upload` folder.

**Comment by chriseth** [comment]
Renamed.

**Review by axic — APPROVED** [review]

**Pull request merged**